Makefile

```diff
@@ -35,19 +37,29 @@ help:
 	@echo
 	@echo 'Configuration options:'
 	@echo
-	@echo '  GITHUB_USER       Which GitHub user account to use for cloning? [default=anonymous]'
-	@echo '  MINIKUBE_DRIVER   Which Minikube driver to use? [default=kvm2]'
-	@echo '  MINIKUBE_PROFILE  Which Minikube profile to use? [default=minikube]'
-	@echo '  TIMECHECK         Checking frequency in seconds when bringing cluster up and down? [default=5]'
-	@echo '  TIMEOUT           Maximum timeout to wait when bringing cluster up and down? [default=300]'
-	@echo '  VENV_NAME         Which Python virtual environment name to use? [default=reana]'
-	@echo '  DEMO              Which demo example to run? [e.g. reana-demo-helloworld; default=several runable examples]'
+	@echo '  GITHUB_USER         Which GitHub user account to use for cloning? [default=anonymous]'
+	@echo '  MINIKUBE_DRIVER     Which Minikube driver to use? [default=kvm2]'
+	@echo '  MINIKUBE_PROFILE    Which Minikube profile to use? [default=minikube]'
+	@echo '  TIMECHECK           Checking frequency in seconds when bringing cluster up and down? [default=5]'
+	@echo '  TIMEOUT             Maximum timeout to wait when bringing cluster up and down? [default=300]'
+	@echo '  VENV_NAME           Which Python virtual environment name to use? [default=reana]'
+	@echo '  DEMO                Which demo example to run? [e.g. reana-demo-helloworld; default=several runable examples]'
+	@echo '  REANA_CLUSTER_MODE  REANA Cluster environment mode [stable|latest|dev, default=latest]'
+	@echo '  DEBUG               Should REANA run with autoreload and debugging capabilities? [default=false]'
 	@echo
 	@echo 'Examples:'
 	@echo
 	@echo '  # how to set up personal development environment:'
 	@echo '  $$ GITHUB_USER=johndoe MINIKUBE_DRIVER=virtualbox make setup clone'
 	@echo
+	@echo '  # how to deploy REANA in development mode, with applicaton'
+	@echo '  # autoreload on code changes and debugging capabilities:'
```

Comment:
For autoreload, perhaps you can emit a very visible help message "Please run minikube mount... in a new terminal" for now.